.github/component_owners.yml

```diff
@@ -82,6 +82,8 @@ components:
     - srprash
     - atshaw43
     - ppittle
+  OpenTelemetry.SpanProcessor.Baggage:
```

Comment:
@MikeGoldsmith @Kielek 
Would this be better made as part of https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Extensions ? Maintaining a new package for a single processor would be bit overkill, I feel!.

Reply:
I like the idea. Forgot about this package (never need to use it).